README.md

```diff
@@ -30,7 +30,7 @@ In build.gradle for your project add:
 allprojects {
     repositories {
         ...
-        maven { url "https://jitpack.io" }
+        maven { url "https://www.jitpack.io" }
```

Comment:
Jitpack sometimes fails for me without the `www` subdomain. Here is some context for it: https://github.com/wordpress-mobile/WordPress-Android/pull/10537

Reply:
Thanks for the background, added a comment there with some thoughts: https://github.com/wordpress-mobile/WordPress-Android/pull/10537#issuecomment-538029366